Automattic/wp-calypso
# Monorepo: Enable lerna, add the root to the lerna config

This allows us to symlink packages in `packages/*` from the root `node_modules`.

Testing Instructions:
* e2e tests should pass
* you should have a symlink from `node_modules/@automattic/tree-select` to the package in `packages/tree-select`
